src/TestFramework/Coverage/JUnit/JUnitTestExecutionInfoAdder.php

```diff
@@ -48,6 +48,9 @@
  */
 class JUnitTestExecutionInfoAdder
 {
+    /**
+     * @var TestFileDataProvider|MemoizedTestFileDataProvider|JUnitTestFileDataProvider
```

Comment:
I don't understand this change: why should `JUnitTestExecutionInfoAdder` care about what implementation can be used here? It's using a `TestFileDataProvider ` that's all it needs to know right?

Reply:
Pure convenience. It's a one click jump for my IDE.

Reply:
It is necessary to add it then? Because we don't do that anywhere else and I think it's more confusing than anything

Reply:
I'd add `@see` in this case, would you mind? 

It really saves my day.

Reply:
isn't going to the interface then the implementation just 1 more click away?

The reason I'm not fond of it is that we have absolutely no mean to keep it in sync and it will get out of sync, adding to the confusion

Reply:
Not in my IDE.

Reply:
That's for sure, but I also don't feel like this is a reliable way to do this and unless we have a tooling to automate this, nobody else in the team is really paying attention to this so it will get out of sync and become incorrect or missing in 99% of the places

I'm happy to let it here (as long as it doesn't add an import, so FQCN might be needed) but just want to be clear: that is neither reliable or sustainable in its current form at least